### DISCUSSION TITLE
Disabling compass marks from scripts does not work

### DISCUSSION THREAD

**Discussion by beltoforion** [Stellarium/stellarium#492] [Q&A] [answered]
Maybe this is just me misinterpreting the documentation. The documentation section for scripting on 0.18 says that the plugin classes are exposed to the scripting engine. (http://stellarium.org/doc/0.18/scripting.html) From this documentation i concluded that i should be able to disable compassmarks with the script command:

CompassMarks.setCompassMarks(false);

However there is no CompassMarks object available. (I also tries CompassMarksMgr). Maybe you can extend the documentation a bit to either remove the section or give the proper syntax for accessing the plugins.

**Comment by alex-w** [comment, marked as answer]
I see from the logs (from your other reports) that Compass plug-in is not loaded at the startup of planetarium (the plugin is disabled). Why you expected the possibility of management for this plugin?

**Comment by beltoforion** [comment]
You are right, that's my fault. I'm closing the issue.

I was under the assumption that with compass marks the "N,S,O,W" labels shown on the horizon were meant. I wanted to get rid of those and was under the assupmtion that this is what the compass marks plugin is used for. Since the markers were there i was under the assumption the plugin was also there.
